spa_ui/self_service/client/app/components/dialog-content/dialog-content.html

```diff
@@ -0,0 +1,69 @@
+<div>
+  <div ng-if="!vm.dialog.label">
+    <h2>No Provisioning Dialog Available.</h2>
+  </div>
+  <div ng-if="vm.dialog.label">
+    <div>
+      <h2 class="text-capitalize no-wrap">{{ ::vm.dialog.label }} </h2>
+      <h4>{{ ::vm.dialog.description }}</h4>
+    </div>
+    <tabset>
+      <tab ng-repeat="dialogTab in vm.dialog.dialog_tabs" heading="{{ dialogTab.label }}">
```

Comment:
Do the label change as the dialog is filled out? Can the `dialogTab.label` be once bound to be removed from the $watch list?